packages/logger/test/end-to-end/compatibility-test-cases.js

```diff
@@ -808,7 +808,16 @@ module.exports = [
 				level: 'error',
 				event: 'FAILED_TO_GET_FT_LIVE_EVENT',
 				message: null,
-				error: {} // Empty object because this is invalid, it says so in the n-logger docs
+				error: {
```

Comment:
**issue:** as well as this compatibility test change we'll also need to add a unit test for this new feature. The reason the build is failing is because, although all the tests are passing, the code coverage has dropped below 100%. Seen in this screen-grab:

<img width="1229" alt="Screenshot 2023-04-26 at 16 40 15" src="https://user-images.githubusercontent.com/138944/234628792-84fa6fa3-ab90-458b-99f3-79feb079dae2.png">

If you run the tests locally you can view code coverage stats in the browser which makes it easy to see what's missing. Run the following to open the report:

```sh
open coverage/lcov-report/logger/lib/logger.js.html
```

(for future reference opening `coverage/lcov-report/index.html` gives the full report)

You'd need to edit the `logger.spec.js` file, copying something like [this code block](https://github.com/Financial-Times/dotcom-reliability-kit/blob/f5622c877821ecf9919fb85643ea4549e96a74c8/packages/logger/test/unit/lib/logger.spec.js#L315-L331) and adapting to provide tests for this new conditional. I'm happy to pair on this if you want to, testing is one of my favourite programming tasks 😁 and some of the mocking we're doing is complex.

Reply:
This looks right to me! It should just be an HTML file so you might be able to open the `coverage` directory in Finder and see what's in there.